source/scss/_pf-basic-tabs.scss

```diff
@@ -0,0 +1,53 @@
+//
+// Tabs
+//
+
+.nav-link {
+    color: $pf-tabs-color;
```

Comment:
I don't think there is a BS variable for the color of a tab in the default state